src/Adapter/Customer/CommandHandler/AbstractCustomerHandler.php

```diff
@@ -74,5 +75,19 @@ protected function assertRequiredFieldsAreNotMissing(Customer $customer)
                 )
             );
         }
+
+        $checkboxErrors = $customer->validateCheckboxRequiredFields();
+
+        if (!empty($checkboxErrors)) {
+            $badFields = array_keys($checkboxErrors);
```

Comment:
You never use `$checkboxErrors`, is it needed to return translated string?

Reply:
You mean I never use the values of `$checkboxErrors` ? Somehow I thought "eyh maybe a module developer could need it" but now it looks like over-engineering, right ?